meta-genivi-dev/recipes-sota/aktualizr/aktualizr_git.bb

```diff
@@ -0,0 +1,43 @@
+SUMMARY = "Aktualizr SOTA Client"
+DESCRIPTION = "SOTA Client written in C++"
+HOMEPAGE = "https://github.com/advancedtelematic/aktualizr"
+SECTION = "base"
+
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=9741c346eef56131163e13b9db1241b3"
+
+inherit cmake systemd
+
+S = "${WORKDIR}/git"
+
+SRCREV = "2c9aabbfd84851309845b4623bb2f20682d47117"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI = " \
+          git://github.com/advancedtelematic/aktualizr \
+          file://0001-rvi_list.h-Fix-rviListGetCount.patch \
+          file://aktualizr.service \
+          "
+
+SYSTEMD_SERVICE_${PN} = "aktualizr.service"
+
+DEPENDS = "boost curl openssl jansson dbus "
+RDEPENDS_aktualizr = "dbus-lib "
```

Comment:
Why do you add dependency with library?
If you really required, please use "RDEPENDS_${PN} = "dbus-lib"